test/Microsoft.TestPlatform.AcceptanceTests/TranslationLayerTests/CustomTestHostTests.cs

```diff
@@ -149,7 +149,7 @@ public void RunAllTestsWithMixedTFMsWillProvideAdditionalInformationToTheDebugge
         var vstestConsoleWrapper = GetVsTestConsoleWrapper();
         var runEventHandler = new RunEventHandler();
         var netFrameworkDll = GetTestDllForFramework("MSTestProject1.dll", DEFAULT_RUNNER_NETFX);
-        var netDll = GetTestDllForFramework("MSTestProject1.dll", "netcoreapp2.1");
+        var netDll = GetTestDllForFramework("MSTestProject1.dll", DEFAULT_RUNNER_NETCORE);
```

Comment:
Both should imho be *_HOST_* variables.

Reply:
https://github.com/microsoft/vstest/pull/3646/files#r919952319

<img width="653" alt="image" src="https://user-images.githubusercontent.com/5735905/180799023-975ceb86-d022-4bce-bba7-b2221d9f73b5.png">


But feel free to fix later, not critical.

Reply:
Done